web/impact/impact/urls.py

```diff
@@ -38,23 +42,20 @@
     UserListView,
     UserOrganizationsView,
 )
-from rest_framework import routers
-from drf_auto_endpoint.router import router as schema_router
-from django.apps import apps
-
 
 accelerator_router = routers.DefaultRouter()
 simpleuser_router = routers.DefaultRouter()
 
 simpleuser_router.register('User', GeneralViewSet, base_name='User')
 
 for model in apps.get_models('impact'):
-    if model._meta.app_label == 'impact':
-        schema_router.register(model, url=model.__name__)
-
+    if model._meta.app_label == 'impact' and not model._meta.auto_created:
+        schema_router.register(model, url=model_name_to_snake(model.__name__))
 
 for model in apps.get_models('accelerator'):
-    if model._meta.app_label == 'accelerator' and hasattr(model, "Meta"):
+    if (model._meta.app_label == 'accelerator' and
+            hasattr(model, "Meta") and
+            not model._meta.auto_created):
```

Comment:
this piece of code is a bit nasty, and should probably be refactored.

Reply:
Chose to follow this style guide: https://www.naftaliharris.com/blog/python-multiline-if-statements/